### ISSUE TITLE
[BUG] Emulator doesn't scale vertically

### ISSUE THREAD

**Issue by jfleckc** [bitfocus/companion#2659]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion
   _(Not sure if this should be classified as a bug, or a feature request - apologies if it's tagged incorrectly)_

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

The emulator scales horizontally, but not vertically.

### Steps To Reproduce

Open up emulator on an Android phone. Save to Home Screen. Open up the newly created web app. The bottom row of buttons is cropped off as the emulator is only horizontally scaling, not vertically. This is reproducible in a browser window on desktop as well as on Android phone.

![Screenshot_20231122-190244](https://github.com/bitfocus/companion/assets/109965087/d7de6386-68ee-4187-86da-2570dde7c00c)

### Expected Behavior

It would be desirable for the emulator to scale vertically as well as horizontally.

### Environment (please complete the following information)

```markdown
- OS: Android
- Browser: Edge and Chrome
- Companion Version: 3.2.0+6545-beta-38a893e5
```

**Comment by Julusian** [comment]
fixed in a9f640f0a8c2c33ac1ba6bc4484ef0ef09d4d1d9

**Comment by jfleckc** [comment]
Tested - works great, thankyou!
